### PR TITLE
Update User Guide

### LINKED ISSUE #96: Update User Guide

### PR DESCRIPTION
**Pull request by Reallyeasy1**
To add in the future: 
1. Assign student to tutorial class command
2. Add more pictures, add in links for better navigation 

Closes #96

### CODE REVIEW

**Comment by Reallyeasy1** [comment]
Delete all demo images

File: docs/UserGuide.md [hunk 1]
```diff
@@ -64,98 +60,172 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+Record a student's attendance in a specific tutorial class, with an optional attendance date (defaults to today if not specified).
+
+![Records attendance](images/recordAttendance.png)
+
+**Command format:** `attend s/STUDENT_ID c/TUTORIAL_CLASS [d/ATTENDANCE_DATE]`
+
+
+Streamline the handling of assignments from creation to tracking:
+
+### Add Assignment
```
Comment (waihongteh) [inline docs/UserGuide.md:165]: Good to see that all assignments' commands are updated in the user guide!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -64,98 +60,152 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+### Add Tutorial
+
+Add a new tutorial session with a specified name and ID.
+
+**Command format:** `addTut tn/TUTORIAL_NAME id/TUTORIAL_ID`  
+**Example:** `addTut tn/CS2103T id/1001`
+
+Maintain accurate records of student attendance across various tutorial classes:
+
+### Record Attendance
```
Comment (tansanchian) [inline docs/UserGuide.md:156]: The title and body are well structured. Well done!

**Review by tansanchian — APPROVED** [review]

File: docs/UserGuide.md [hunk 3]
```diff
@@ -180,26 +230,42 @@ _Details coming soon ..._
...
-Action     | Format, Examples
------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**Add**    | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear**  | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List**   | `list`
-**Help**   | `help`
+| Action            | Format, Examples                                                                                                                                                       |
```
Comment (SamsonChew) [inline docs/UserGuide.md:247]: Some of the command format is wrong
- Add student command will need to be in the format of add n/STUDENT_NAME s/STUDENT_ID c/TUTORIAL_CLASS_ID
- Edit student command will need to be in the format of edit INDEX [n/STUDENT_NAME] [s/STUDENT_ID] [c/TUTORIAL_CLASS_ID]

File: docs/UserGuide.md [hunk 4]
```diff
@@ -15,30 +15,26 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+    * `list` : Lists all contacts.
+    * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
```
Comment (Fui03) [inline docs/UserGuide.md:32]: Could you please verify and update the latest add command?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -15,30 +15,26 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Ensure you have Java `17` or above installed on your Computer.
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
```
Comment (Fui03) [inline docs/UserGuide.md:20]: Could you please verify and update the correct URL?

**Review by Fui03 — COMMENTED** [review]
Change the comment in the next pull request!

**Pull request merged**